### ISSUE TITLE
Unable to connect to Westend - Error: Cannot construct unknown type WeakVec

### ISSUE THREAD

**Issue by kenhaise** [polkadot-js/api#3612]
Getting the following error when trying to connect to the westend node:
`Unable to decode storage balances.locks: entry 1:: createType(WeakVec):: Cannot construct unknown type WeakVec
`

<img width="928" alt="image" src="https://user-images.githubusercontent.com/83870334/120481666-f4573500-c3cd-11eb-9616-1c42f1eaa538.png">

```
const wsProvider = new WsProvider("wss://westend-rpc.polkadot.io");
const api = await ApiPromise.create({ provider: wsProvider });
```
versions that i am using:
```
"@polkadot/api": "^4.12.1",
"@polkadot/extension-dapp": "^0.35.1",
"@polkadot/react-identicon": "^0.58.1",
"@polkadot/rpc-core": "^4.12.1",
"@polkadot/util": "^6.1.1",
"@polkadot/util-crypto": "^6.1.1",

```

**Comment by jacogr** [comment]
Already available in https://github.com/polkadot-js/api/pull/3607

**Comment by kenhaise** [comment]
thank you fixed after upgading to 4.12.2-9
